pkg/acs/calacs/lib/reffiles.c

```diff
@@ -128,7 +128,7 @@ RefFileInfo *ref   io: the list of reference info
     if (!ref)
         return;
 
-	RefFileInfo *current, *next;
+    RefFileInfo *current, *next;
 
```

Comment:
I agree to moving the declaration to the same indented position as the "if", but I am not sure why the code following this line was not also adjusted.  My statement is true for all of the reffiles.c programs.

Reply:
They are both indented, the `if` with 4 ws and this dec with a tab. GitHub renders these differently for observation clarity. I could have changed the following lines of code for consistency but this would then link these lines to this PR via git blame. This is something I wanted to avoid. Changing the dec line was that minimally required to remove the compiler warning.

Reply:
OK - understood.

Reply:
I could have used a tab to indent the intended changed lines which would have also solved the issue, but then ws is better than a tab.